scripts/long-description/long_description.py

```diff
@@ -0,0 +1,86 @@
+import json, requests, time, sys
+from bs4 import BeautifulSoup
+from configuration import *
+
+# Get an access token for authenticating API requests
+def get_access_token():
+    post_data = {'username': td_api_user, 'password': td_api_pass}
+    url = td_api_url + "/auth"
+    request = requests.post(url, json=post_data)
+    if request.status_code is not 200:
+        print("Error: Unable to get access token.")
+        print(request.text)
+        sys.exit(1)
+    
+    return request.text
+
+# Find span html tags and create a dict of their id and value
+def get_parsed_html(raw_html):
+    parsed_html = BeautifulSoup(raw_html, "html.parser")
+    span_dict = {}
+    spans = parsed_html.find_all('span')
+
+    for span in spans:
+        span_id = span.get('id')
+        if span_id:
+            span_contents = span.contents
+            span_full_string = ""
+
+            # need a better way to get the literal contents of tag. span.text strips out html tags 
+            for content in span_contents:
+                span_full_string += str(content)
```

Comment:
concatenating to a string in a loop is quadratic in the length of the string. better to construct a list and join at the end, like `"".join([str(content) for content in span.contents])`. probably not a big deal here, but it's something to watch out for.